### PR TITLE
Spelling & Formatting & Other

### PR DESCRIPTION
**Pull request by tschmidtb51**
- fix spelling mistakes
- correct format issues
- add OpenPGP key strength comment
- adopt review comments

### CODE REVIEW

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 1]
```diff
@@ -4489,9 +4487,9 @@ The relevant path for this test is:
 
 ### 6.2.8 Use of MD5 as the only Hash Algorithm
 
-It must be tested that the hash algorithm `md5` is not the only one present.
+It MUST be tested that the hash algorithm `md5` is not the only one present.
 
-> Since collision attacks exist for MD5 such value should be accompanied by a second cryptographically stronger hash. This will allow users to double check the results.
+> Since collision attacks exist for MD5 such value SHOULD be accompanied by a second cryptographically stronger hash. This will allow users to double check the results.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:4492]: Same here: preferred rephrase to not have the shouting words, if not convincing keep lowercase, or if applicable augment the text part to normative.

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 2]
```diff
@@ -5470,7 +5472,7 @@ The file `aggregator.json` MUST be present and valid according to the JSON schem
 > * https://psirt.domain.tld/advisories/aggregator/csaf/aggregator.json
 > * https://domain.tld/security/aggregator/csaf/aggregator.json
 
-The file `aggregator.json` SHOULD only list the latest version of the metadata of a CASF provider.
+The file `aggregator.json` SHOULD only list the latest version of the metadata of a CSAF provider.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:5475]: 🙈

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 3]
```diff
@@ -6020,7 +6022,7 @@ A CSAF asset matching system satisfies the "CSAF asset matching system" conforma
     * when a new asset is inserted (for this asset)
     * when the Major version in a CSAF document with semantic versioning changes (for this CSAF document)
     > These also apply if more than one CSAF document or asset was added. To reduce the computational efforts the runs can be pooled into one run which fulfills all the tasks at once (batch mode).
-  > Manually and automatically triggered runs should not be pooled.
+  > Manually and automatically triggered runs SHOULD not be pooled.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:6025]: Ibid rephrase, lowercase, or augment text part to normative

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 4]
```diff
@@ -6425,7 +6428,7 @@ A string with format `uri` should not have a length greater than 20000. This app
 
 A string which is an enum has a fixed maximum length given by its longest value.
 
-> Later versions of CSAF might add, modify or delete possible value which could change the longest value. Therefore, this sizes should not be implemented as fixed limits if forward compatibility is desired.
+> Later versions of CSAF might add, modify or delete possible value which could change the longest value. Therefore, this sizes SHOULD NOT be implemented as fixed limits if forward compatibility is desired.
```
Comment (sthagen) [inline csaf_2.0/prose/csaf-v2-editor-draft.md:6431]: Ibid rephrase, lowercase, or augment text part to normative

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 2 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md:5475]: Same here.

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 5]
```diff
@@ -2794,12 +2792,12 @@ This profile SHOULD be used to provide information which are **not related to a
...
 A CSAF document SHALL fulfill the following requirements to satisfy the profile "Informational Advisory":
 
-* The following elements must exist and be valid:
+* The following elements MUST exist and be valid:
   * all elements required by the profile "Generic CSAF".
   * `/document/notes` with at least one item which has a `category` of `description`, `details`, `general` or `summary`
     > Reasoning: Without at least one note item which contains information about the "issue" which is the topic of the advisory it is useless.
   * `/document/references`
-    > This should be used to refer to one or more documents or websites which provide more details about the issue or its remediation (if possible). This could be a hardening guide, a manual, best practices or any other helpful information.
+    > This SHOULD be used to refer to one or more documents or websites which provide more details about the issue or its remediation (if possible). This could be a hardening guide, a manual, best practices or any other helpful information.
```
[the remark this answers is not in the document]
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: Addressed in [6356a59](https://github.com/oasis-tcs/csaf/pull/449/commits/6356a59f8647b9c1995408defb628b478783b622).

File: csaf_2.0/prose/csaf-v2-editor-draft.md [hunk 6]
```diff
@@ -3741,9 +3739,9 @@ The relevant paths for this test are:
 
 ### 6.1.26 Prohibited Document Category Name
 
-It must be tested that the document category is not equal to the (case insensitive) name of any other profile than "Generic CSAF". This does not differentiate between underscore, dash or whitespace. This test does only apply for CSAF documents with the profile "Generic CSAF". Therefore, it must be skipped if the document category matches one of the values defined for the profile other than "Generic CSAF".
+It MUST be tested that the document category is not equal to the (case insensitive) name of any other profile than "Generic CSAF". This does not differentiate between underscore, dash or whitespace. This test does only apply for CSAF documents with the profile "Generic CSAF". Therefore, it MUST be skipped if the document category matches one of the values defined for the profile other than "Generic CSAF".
 
-> For CSAF 2.0, the test must be skipped for the following values in `/document/category`:
+> For CSAF 2.0, the test MUST be skipped for the following values in `/document/category`:
```
[the remark this answers is not in the document]
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: Addressed in [6356a59](https://github.com/oasis-tcs/csaf/pull/449/commits/6356a59f8647b9c1995408defb628b478783b622).

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 1 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: Addressed in [6356a59](https://github.com/oasis-tcs/csaf/pull/449/commits/6356a59f8647b9c1995408defb628b478783b622).

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 3 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: Addressed in [6356a59](https://github.com/oasis-tcs/csaf/pull/449/commits/6356a59f8647b9c1995408defb628b478783b622).

File: csaf_2.0/prose/csaf-v2-editor-draft.md (hunk 4 above)
Comment (tschmidtb51) [reply csaf_2.0/prose/csaf-v2-editor-draft.md]: Addressed in [6356a59](https://github.com/oasis-tcs/csaf/pull/449/commits/6356a59f8647b9c1995408defb628b478783b622).

**Review by sthagen — APPROVED** [review]
Tank you, @tschmidtb51 - this looks good for me to merge.

**Pull request merged**